### PR TITLE
Move to CaaS CRD

### PR DESCRIPTION
**Pull request by mkjpryor**
Includes removing AWX components by setting a flag, `awx_purge=yes`

Supersedes #62

### CODE REVIEW

File: roles/azimuth_caas_operator/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  playbook: "{{ azimuth_caas_stackhpc_workstation_playbook }}"
+  extraVars: "{{ azimuth_caas_stackhpc_workstation_extra_vars }}"
+
+
+# Indicates if the StackHPC repo2docker appliance should be enabled
+azimuth_caas_stackhpc_repo2docker_enabled: "{{ azimuth_clusters_enabled }}"
+# The git URL for the StackHPC repo2docker appliance
+azimuth_caas_stackhpc_repo2docker_git_url: https://github.com/stackhpc/caas-repo2docker.git
+# The git version for the StackHPC repo2docker appliance
+azimuth_caas_stackhpc_repo2docker_git_version: 8385160f69646e615a51173e107e20e2485e2457
```
Comment (JohnGarbutt) [inline roles/azimuth_caas_operator/defaults/main.yaml:197]: These should be tags, but we have a task to go fix that I guess.

File: roles/azimuth_caas_operator/defaults/main.yaml [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+azimuth_caas_cluster_templates_default: >-
+  {{-
+    {} |
+      combine(
+        { "slurm": azimuth_caas_stackhpc_slurm_appliance_template }
+        if azimuth_caas_stackhpc_slurm_appliance_enabled
+        else {}
+      ) |
+      combine(
+        { "workstation": azimuth_caas_stackhpc_workstation_template }
```
Comment (JohnGarbutt) [inline roles/azimuth_caas_operator/defaults/main.yaml:260]: I think we should include the workstation with ssh template in here as well.

**Review by JohnGarbutt — APPROVED** [review]
I have ideas, but lets just get this in now.

File: roles/azimuth_caas_operator/defaults/main.yaml (hunk 1 above)
Comment (mkjpryor) [reply roles/azimuth_caas_operator/defaults/main.yaml:197]: That will happen as part of the release

**Review by JohnGarbutt — APPROVED** [review]
cool

**Pull request merged**